packages/system-tests/package.json

```diff
@@ -25,6 +25,8 @@
     "istanbul": "^0.4.5",
     "mkdirp": "0.5.1",
     "mocha": "3.2.0",
+    "mocha-junit-reporter": "^1.17.0",
```

Comment:
These versions are different than the other packages. Is there a feature/bug fix in the newer version that we need for system tests?

Reply:
No need for the later version. Downgraded it to 1.13